src/_locales/en_US/messages.json

```diff
@@ -508,5 +508,41 @@
                 "example": "Facebook Like"
             }
         }
+    },
+    "sync_intro": {
+        "message": "Cloud sync:<ul><li>Requires Firefox/Chrome Sync</li><li>Upload overwrites any existing Privacy Badger data in the cloud</li><li>Download combines the lists of sites where your Badger is disabled</li></ul>",
+        "description": "A brief explanation of how syncing works. Shown above the upload/download cloud data buttons under the Manage Data options page tab."
+    },
+    "upload": {
+        "message": "Upload",
+        "description": "Button label for uploading data to the cloud on Manage Data options page tab."
+    },
+    "upload_cloud": {
+        "message": "Export disabled sites to cloud",
```

Comment:
I think changing the phrasing here from "import/export" to "upload/download" would be more consistent and easier to understand.

Reply:
@ghostwords suggested import/export to be consistent with the other pair of user data buttons.

Reply:
Oh ok! didn't see that conversation. I'll defer to his judgment here.